src/lib/orionTypes/OrionValueType.h

```diff
@@ -34,16 +34,23 @@ namespace orion
 /* ****************************************************************************
 *
 * ValueType - 
+*
+* Note ValueTypeNotGiven is only used by NGSIv1 parsing logic. In NGSIv2 parsing,
+* attribute or metatadata value is implicit if ommitted (in particular, it
```

Comment:
typo: omitted

Reply:
Fixed in e85892cdb28f2f510fd2b0a38cacbb43082f9613